andela/ah-codeblooded-frontend
# #160577588 Enable users to favourite and unfavourite articles

**What does this PR do?**

Enable users to favourite articles that they deem fancy and unfavourite them whenever they want.

**Description of Task to be completed?**

Articles that tickle our fancy are worth favouriting and those deemed distasteful are worth unfavouriting.

**How should this be manually tested?**

To test this manually:
1. Open this PR's review app.
2. Click on an article to open it.
3. Scroll to the bottom of the page.
4. lick on the heart shaped icon if you like the article. It should change colour. Click again to unfavorite.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

Users should be able to favourite articles: [https://www.pivotaltracker.com/story/show/160577588]

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-11-22 at 10 12 29" src="https://user-images.githubusercontent.com/26184534/48896474-c128a800-ee58-11e8-8843-6508b1d46882.png">
<img width="1440" alt="screen shot 2018-11-22 at 10 32 50" src="https://user-images.githubusercontent.com/26184534/48896480-c71e8900-ee58-11e8-9264-6747e5173777.png">

**Questions:**

N/A
